tests/test_cli.py

```diff
@@ -0,0 +1,140 @@
+from unittest import mock
+from io import StringIO
+import pytest
+
+import buku
+
+
+@pytest.fixture
+def stdin(monkeypatch):
+    with monkeypatch.context():
+        monkeypatch.setattr('sys.stdin', (buffer := StringIO()))
+        yield buffer
+
+@pytest.fixture
+def BukuDb():
+    with mock.patch('buku.BukuDb') as cls:
+        cls.return_value.close_quit.side_effect = SystemExit
+        yield cls
+
+@pytest.fixture
+def bdb(BukuDb):
+    yield BukuDb.return_value
```

Comment:
The `BukuDb` fixture can be used when testing invocation of constructor/static methods; otherwise `bdb` is more convenient